.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
This file is the configuration for [ESLint](https://eslint.org/), a tool that finds and points out errors and warnings in your code. Most of this was automatically generated from `npm init @eslint/config` when I followed the [installation steps](https://www.npmjs.com/package/eslint#installation-and-usage).